config/v1/0000_10_config-operator_01_node.crd.yaml

```diff
@@ -0,0 +1,108 @@
+          type: object
+          required:
+            - spec
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              type: string
+            kind:
+              description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: spec holds user settable values for configuration
+              type: object
+              properties:
+                cgroupMode:
+                  description: CgroupMode determines the cgroups version on the node
+                  type: string
+                  enum:
+                    - v1
+                    - v2
+                    - ""
+                workerLatencyProfile:
+                  description: WorkerLatencyProfile determins the how fast the kubelet is updating the status and corresponding reaction of the cluster
+                  type: string
+                  enum:
+                    - Default
+                    - MediumUpdateAverageReaction
+                    - LowUpdateSlowReaction
+            status:
+              description: status holds observed values.
+              type: object
+              properties:
+                workerLatencyProfileStatus:
+                  description: WorkerLatencyProfileStatus provides the current status of WorkerLatencyProfile
+                  type: object
+                  properties:
+                    conditions:
+                      description: conditions describes the state of the WorkerLatencyProfile and related components (Kubelet or Controller Manager or Kube API Server)
+                      type: array
+                      items:
+                        description: "Condition contains details for one aspect of the current state of this API Resource. --- This struct is intended for direct use as an array at the field path .status.conditions.  For example, type FooStatus struct{     // Represents the observations of a foo's current state.     // Known .status.conditions.type are: \"Available\", \"Progressing\", and \"Degraded\"     // +patchMergeKey=type     // +patchStrategy=merge     // +listType=map     // +listMapKey=type     Conditions []metav1.Condition `json:\"conditions,omitempty\" patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"` \n     // other fields }"
```

Comment:
needs an update?

Reply:
This `description` is relying on description message from upstream k8s metav1.Condition, whenever I change this description manually running `make generate-with-container` reverts it back to this very message.

Reply:
Ah sorry, I saw  `\"Progressing\", and \"Degraded\"` and thought it was related to our conditions. On a second look this is exact copy from metav1.Condition. So we can keep it as is.

Reply:
Alright, thanks!